### PR TITLE
FA-93 createAccountCore Modules added

### PR DESCRIPTION
**Pull request by brosscode**

### CODE REVIEW

File: core/src/main/java/com/brandon/di/NetworkModule.kt [hunk 1]
```diff
@@ -43,4 +44,11 @@ object NetworkModule {
         return forzenDatabase.forzenDao()
     }
 
+    @Provides
+    fun providesConnectivityManager(@ApplicationContext context: Context): ConnectivityManager {
+        return context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
+    }
+
+    fun ConnectivityManager.isConnected(): Boolean = activeNetworkInfo?.isConnected() ?: false
```
Comment (ForzenAcademy) [inline core/src/main/java/com/brandon/di/NetworkModule.kt]: I don't think it makes sense for this to be in a Module that's meant for DI. You can make an Extensions file or something similar to put these in

File: createAccountCore/build.gradle [hunk 2]
```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+    id 'kotlin-kapt'
+    id 'com.google.dagger.hilt.android'
+}
+
+android {
+    namespace 'com.brandon.createaccountcore'
+    compileSdk 32
```
Comment (ForzenAcademy) [inline createAccountCore/build.gradle]: You should be on 33

File: createAccountCore/build.gradle [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
```
Comment (ForzenAcademy) [inline createAccountCore/build.gradle]: You should be on VERSION_11

File: createAccountCore/build.gradle [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
```
Comment (ForzenAcademy) [inline createAccountCore/build.gradle]: 11

File: createAccountCore/proguard-rules.pro [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (ForzenAcademy) [inline createAccountCore/proguard-rules.pro]: You don't need this file or the `consumer-rules` file

File: createAccountCore/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ForzenAcademy) [inline createAccountCore/src/main/AndroidManifest.xml:1]: Condense this file

File: createAccountCore/src/main/java/com/brandon/createaccountcore/data/CreateUserRepository.kt [hunk 7]
```diff
@@ -1,4 +1,4 @@
-package com.brandon.forzenbook.repository
+package com.brandon.createaccountcore.data
```
Comment (ForzenAcademy) [inline createAccountCore/src/main/java/com/brandon/createaccountcore/data/CreateUserRepository.kt]: Your package name is busy. You can make it `createaccount.core.data` to break it up

File: core/src/main/java/com/brandon/di/NetworkModule.kt (hunk 1 above)
Comment (brosscode) [reply core/src/main/java/com/brandon/di/NetworkModule.kt]: created a extensions file and moved it there

File: createAccountCore/build.gradle (hunk 2 above)
Comment (brosscode) [reply createAccountCore/build.gradle]: updated

File: createAccountCore/build.gradle (hunk 3 above)
Comment (brosscode) [reply createAccountCore/build.gradle]: updated

File: createAccountCore/build.gradle (hunk 4 above)
Comment (brosscode) [reply createAccountCore/build.gradle]: updated to 11

File: createAccountCore/src/main/AndroidManifest.xml (hunk 6 above)
Comment (brosscode) [reply createAccountCore/src/main/AndroidManifest.xml:1]: condensed

File: createAccountCore/src/main/java/com/brandon/createaccountcore/data/CreateUserRepository.kt (hunk 7 above)
Comment (brosscode) [reply createAccountCore/src/main/java/com/brandon/createaccountcore/data/CreateUserRepository.kt]: Fixed this and used `createaccount.core.data` as suggested

**Review by ForzenAcademy — APPROVED** [review]

File: createAccountCore/proguard-rules.pro (hunk 5 above)
Comment (brosscode) [reply createAccountCore/proguard-rules.pro]: Deleted.

**Pull request merged**